UnityProject/Assets/Scripts/Messages/Server/FinishedAddedObserver.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using Messages.Server;
+using Mirror;
+using UnityEngine;
+
+public class FinishedAddedObserver : ServerMessage<FinishedAddedObserver.NetMessage>
```

Comment:
```suggestion
public class FinishedAddedObserverMessage : ServerMessage<FinishedAddedObserver.NetMessage>
```
also add a namespace